project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version = 0.13.12
```

Comment:
No, i don't have a particular problem with the fixed version, but as far as I understand it's used to force a minimum required sbt-version for your build. As `hidden-args` doesn't really requires this, I removed the fixed version.